### ISSUE TITLE
Alerting support

### ISSUE THREAD

**Issue by cosmos1978** [leoswing/comparequeries-datasource-rc#14]
At the moment alerting is not supported on this datasource.

I would be really interesting to have alerting working for this datasource.
I would allow for alerts on timeshifted data. so when some data falls 20% below the amount of last week the alert is triggered.

Thanks,

**Comment by leoswing** [comment]
> At the moment alerting is not supported on this datasource.
> 
> I would be really interesting to have alerting working for this datasource. I would allow for alerts on timeshifted data. so when some data falls 20% below the amount of last week the alert is triggered.
> 
> Thanks,

Yep thanks for your suggestion, I get it. Maybe I need a few days to work on it when free.

**Comment by eraserhead0705** [comment]
+1

@leoswing any update? This is a major use case for this plugin imho.

**Comment by leoswing** [comment]
Sorry for late response, if alerting needs for this plugin, it would need a breaking change for the ui and changes to a backend plugin with datasource setting. @eraserhead0705 @cosmos1978 

Not sure whether it will lead to a problem for using the plugin

**Comment by leoswing** [comment]
@eraserhead0705 @cosmos1978 Hi guys，would you please help to review the solution in [alerting issue](https://github.com/leoswing/comparequeries-datasource-rc/issues/13) ，and check whether it satisfied your scenario？

I'm working on the plugin with backend data alert support, and now it's in testing mode. Could you guys help to check wheter it's what you need?

**Comment by leoswing** [comment]
Plugin with alerting support now has been released！Version 2.1.0 has been published in Grafana plugin market and publish submission has been approved.

Please use the new version to experience the new feature ~ @eraserhead0705 @cosmos1978

**Comment by cosmos1978** [comment]
> Plugin with alerting support now has been released！Version 2.1.0 has been published in Grafana plugin market and publish submission has been approved.
> 
> Please use the new version to experience the new feature ~ [@eraserhead0705](https://github.com/eraserhead0705) [@cosmos1978](https://github.com/cosmos1978)

Many thanks. I will start testing this next week

**Comment by leoswing** [comment]
@eraserhead0705 @cosmos1978 Issue close due to completed, Version 2.1.1 would help you build with the alert, and a github star would be appreciated if it helps you a lot ~
